src/vm/method.hpp

```diff
@@ -3277,15 +3310,31 @@ class InstantiatedMethodDesc : public MethodDesc
     }
 #endif // FEATURE_COMINTEROP
 
+    PTR_DictionaryLayout GetDictLayout()
```

Comment:
I think this should be called GetDictLayoutRaw to avoid confusion with the regular GetDictionaryLayout method.